libs/gl-client/src/signer/.#mod.rs

```diff
@@ -0,0 +1 @@
+erik@erik-tp.561835:1692270444
```

Comment:
You probably don't want to commit this :-) But it's good to see a fellow emacs user :-)

Reply:
I think you might have forgotten to force-push, the PR still contains the file :-)